include/deal.II/base/point.h

```diff
@@ -307,7 +307,7 @@ class Point : public Tensor<1, dim, Number>
 
   /**
    * Return the Euclidean distance of <tt>this</tt> point to the point
-   * <tt>p</tt>, i.e. the <tt>l_2</tt> norm of the difference between the
+   * <tt>p</tt>, i.e. the $\ell_2$ norm of the difference between the
```

Comment:
No, that's yet another space: The space of infinite dimensional vectors: https://en.wikipedia.org/wiki/Hilbert_space#Sequence_spaces. It needs to be `$l_2$`.